caas/containers.go

```diff
@@ -3,67 +3,77 @@
 
 package caas
 
-import (
-	"github.com/juju/errors"
-	"gopkg.in/yaml.v2"
-)
-
-// ContainerPort defines the attributes used to configure
-// an open port for the container.
-type ContainerPort struct {
-	ContainerPort int    `yaml:"container-port"`
-	Protocol      string `yaml:"protocol"`
-}
+import "github.com/juju/errors"
 
 // FileSet defines a set of files to mount
 // into the container.
 type FileSet struct {
-	Name      string            `yaml:"name"`
-	MountPath string            `yaml:"mount-path"`
-	Files     map[string]string `yaml:"files"`
+	Name      string            `yaml:"name" json:"name"`
+	MountPath string            `yaml:"mountPath" json:"mountPath"`
```

Comment:
I wonder why camelCase change though? Why not hyphenated?

Reply:
This is what kubernetes uses so I wanted to be consistent with what people expect to use already